src/app/shared/language/language.data.ts

```diff
@@ -0,0 +1,4 @@
+export const languageList: string[] = [
+  'English',
```

Comment:
en, kh

Reply:
create a json file where you translate `'LANGUAGE_NAME' : { 'EN': 'English', 'KH': 'Khmer' }`